creator-node/src/middlewares/readOnly/readOnlyMiddleware.test.js

```diff
@@ -0,0 +1,60 @@
+const { readOnlyMiddleware, readOnlyMiddlewareHelper } = require('./readOnlyMiddleware')
+const assert = require('assert')
+const config = require('../../config')
+const { resFactory, loggerFactory } = require('../../../test/lib/reqMock')
```

Comment:
TODO add file from second PR here

Reply:
fixed. and i also fixed the run-tests.sh to error on the tests actually failing which it was not doing before

Reply:
https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/5573/workflows/3bb7a847-28af-4e8c-a232-378738d6ebc4/jobs/41434